it/uk/gov/hmrc/personaldetailsvalidation/PersonalDetailsValidationISpec.scala

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmrc.personaldetailsvalidation
+
+import java.util.UUID.randomUUID
+
+import play.api.http.Status._
+import play.api.libs.json.Json
+import play.mvc.Http.HeaderNames.LOCATION
+import uk.gov.hmrc.support.BaseIntegrationSpec
+
+class PersonalDetailsValidationISpec extends BaseIntegrationSpec {
+
+  "POST /personal-details-validations" should {
+    "create a personal-details-validation resource" in new Setup {
+      val personalDetails =
+        """
+          |{
+          |   "firstName":"Joe",
+          |   "lastName":"Ferguson",
+          |   "nino":"AA000003D",
+          |   "dateOfBirth":"2017-12-31"
+          |}
+        """.stripMargin
+      val createResponse = sendCreateValidationResourceRequest(personalDetails).futureValue
+      createResponse.status mustBe CREATED
+      val Some(resourceUrl) = createResponse.header(LOCATION)
+
+      val getResponse = wsUrl(resourceUrl).get().futureValue
+      getResponse.status mustBe OK
+      resourceUrl must endWith ((getResponse.json \ "id").as[String])
+      (getResponse.json \ "validationStatus").as[String] mustBe "success"
+      (getResponse.json \ "personalDetails").toOption must contain(Json.parse(personalDetails))
```

Comment:
Can be simplified to `(getResponse.json \ "personalDetails").as[JsValue] mustBe Json.parse(personalDetails)`